common/lib/dependabot/shared_helpers.rb

```diff
@@ -374,12 +374,34 @@ def self.configure_git_to_use_https(host)
 
     sig { params(path: String).void }
     def self.reset_git_repo(path)
-      Dir.chdir(path) do
-        run_shell_command("git reset HEAD --hard")
-        run_shell_command("git clean -fx")
+      if lfs_enabled?(path) 
```

Comment:
Why do we need 3 different methods in 3 different places to check if lfs is enabled?

Reply:
Because we have three different files in common that each call the git command in a different way and none of them refer to the others.  To make matters worse, some code never even bothers to use any of them.

Reply:
Seems like the could all still share a single, shared way to check if lfs is enabled?